ReportPortal.NUnitExtension/ReportPortalListener.Suite.cs

```diff
@@ -120,7 +120,7 @@ private void FinishSuite(XmlDocument xmlDoc)
                             finishSuiteRequest.Description = description.Attributes["value"].Value;
                         }
 
-                        var eventArg = new TestItemFinishedEventArgs(Bridge.Service, finishSuiteRequest, _flowItems[id].Reporter);
+                        var eventArg = new TestItemFinishedEventArgs(Bridge.Service, finishSuiteRequest, _flowItems[id].Reporter, xmlDoc.InnerXml);
```

Comment:
`OuterXml` covers more than `InnerXml`

Reply:
Done

Reply:
But IMHO for XmlDocument this should be the same